forgerock-authenticator/src/test/java/org/forgerock/android/auth/PushRequestInterceptor.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth;
+
+import static org.forgerock.android.auth.Action.PUSH_AUTHENTICATE;
+import static org.forgerock.android.auth.Action.PUSH_REGISTER;
+
+import android.net.Uri;
+
+import androidx.annotation.NonNull;
+
+public class PushRequestInterceptor implements FRRequestInterceptor<Action> {
```

Comment:
move this class to the same test class , because this is not shared across multiple tests.

Reply:
done